docs/docs/metrics.md

```diff
@@ -4,112 +4,89 @@ title: Metrics
 
 # Metrics
 
-Marathon currently uses the [Codahale/Dropwizard Metrics](https://github.com/dropwizard/metrics). You can query
-the current metrics via the `/metrics` HTTP endpoint or configure reporting these metrics periodically to
+Marathon currently uses [Codahale/Dropwizard Metrics](https://github.com/dropwizard/metrics). You can query
+the current metrics via the `/metrics` HTTP endpoint or configure the metrics to report periodically to:
 
 * graphite via `--reporter_graphite`.
 * datadog via `--reporter_datadog`.
-* statsd via `--reporter_datadog` (not a typo, the datadog reports supports statsd).
+* statsd via `--reporter_datadog` (datadog reports supports statsd).
 
-For the specific syntax check out the
+For the specific syntax see the
 [command line flag metrics]({{ site.baseurl }}/docs/command-line-flags.html#metrics-flags) section.
 
 ## Stability of metric names
 
-Even though we try to prevent unnecessary disruptions, we do not provide any stability guarantees for metric
-names between major/minor releases. We will not change the name of a metric non-method-call (see below) metric
-in a patch release if this is not absolutely required to fix a production issue (very unlikely).
+Although we try to prevent unnecessary disruptions, we do not provide stability guarantees for metric names between major and minor releases.
+
+We will not change the name of a metric non-method-call (see below) metric in a patch release if this is not required to fix a production issue, which is very unusual.
 
 ## Metric names
 
-All metric names have to prefixed by the prefix that you configure and are subject to modification by statsd and
-the like. For example, if we write that the name of a metric is "service.mesosphere.marathon.uptime", it might
-be available under "stats.gauges.marathon_test.service.mesosphere.marathon.uptime" in your configuration.
+All metric names have to prefixed by a prefix that you specify and are subject to modification by statsd and the like. For example, if we write that the name of a metric is `service.mesosphere.marathon.uptime`, it might be available under `stats.gauges.marathon_test.service.mesosphere.marathon.uptime` in your configuration.
```

Comment:
Marathon can report metrics to different time series databases. What those tools do with the data is out of control of Marathon. I am not sure, if tools like statsd mess with the name - I don't think so.